test/integration/targets/command_shell/tasks/main.yml

```diff
@@ -282,6 +320,22 @@
     path: "{{ output_dir_test }}/afile.txt"
     state: file
 
+# changes
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'win' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
```

Comment:
Same comment as above: verifying before the tasks run seem to be verifying the result of previous tests, not tests related to `changes`.